pkg/abi/abidecode_test.go

```diff
@@ -285,7 +284,7 @@ func TestExampleABIDecode7(t *testing.T) {
 func TestExampleABIDecodeTupleDifferentOrder(t *testing.T) {
 
 	f := &Entry{
-		Name: "coupon",
+		// Name: "coupon",
```

Comment:
Lots of "Names" commented out in this file. Should they be deleted?

Reply:
yeah - it's linter cruft - will remove